Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Nikos Baxevanis</authors>
+        <owners>Nikos Baxevanis</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.1" />
```

Comment:
Is there a particular reason for requiring FakeItEasy version 2.1? Or would everything also work with FakeItEasy version 2.0?